openlibrary/i18n/fr/messages.po

```diff
@@ -3425,13 +3425,12 @@ msgid "Digital shelves organized like a physical library"
 msgstr "Étagères numériques organisées comme dans une bibliothèque"
 
 #: home/welcome.html:68
-#, fuzzy
 msgid "Try Fulltext Search"
-msgstr "Recherche en plein texte"
+msgstr "Essayez la recherche en plein texte"
```

Comment:
The approach looks good to me! Could you rebase? These changes were from before I believe, and are preventing me from deploying to testing.openlibrary.org :)

Reply:
@cdrini Yes :)